requirements.txt

```diff
@@ -4,6 +4,7 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+appdirs==1.4.3            # via setuptools
```

Comment:
this PR have more changes than others... probably because we updated this stack long time ago.